### DISCUSSION TITLE
How can I show that I am still employed?

### DISCUSSION THREAD

**Discussion by nf9k** [rendercv/rendercv#133] [Q&A] [answered]
I did not see a good way to indicate I'm still employed?  I've always used, "Current," but other synonyms could work, too.

**Comment by sinaatalay** [comment, marked as answer]
There are two ways:

- Either not providing `end_date`.
- Or setting `end_date` to present.

Then the output will be "present".

However, if you would like to use "current" instead of "present" you could append this to your YAML file:

```yaml
locale_catalog:
  present: current
```

See [the `locale_catalog` field's documentation](https://docs.rendercv.com/user_guide/structure_of_the_yaml_input_file/#locale_catalog-field) for details.

Also make sure to use JSON Schema to find out available options, as described [here](https://docs.rendercv.com/user_guide/structure_of_the_yaml_input_file).
